src/app/utils/date.ts

```diff
@@ -31,3 +31,17 @@ export const createQueryWithDateParam = (
   // Start and end date is missing, return empty object.
   return {}
 }
+
+/**
+ * Returns start of day. If date is not given, the start of day will be
+ * calculated based on today's date.
+ *
+ * @param date the date used to calculate the starting day. Defaults to today's
+ * date.
+ */
+export const getStartOfDay = (date?: Date): Date => {
+  if (!date) {
+    date = new Date()
+  }
+  return moment.tz(date, 'Asia/Singapore').startOf('day').toDate()
+}
```

Comment:
Can I also clarify what we are using this function for? If we want to constantly set the timezone to Singapore, we should also name the function accordingly bah

Reply:
the utility function itself should be self contained. refactor a little [here](https://github.com/opengovsg/FormSG/pull/6467/files/5725ee19be452e4485d49df8f98c2d4e10169d6f). let me know if it's still unclear.